backend/Makefile

```diff
@@ -0,0 +1,23 @@
+default: start
+
+start:
+	python3 manage.py runserver
+
+install:
+	pip install -r requirements.txt
+
+test:
+	python3 manage.py test
+
+docker:
+	docker-compose up &
```

Comment:
Is `&` a typo? Never seen that before. https://docs.docker.com/compose/reference/up/

Reply:
Nop! `&` is a pure Unix construct to run a background job, e.g., https://www.ibm.com/support/knowledgecenter/SSLTBW_2.2.0/com.ibm.zos.v2r2.bpxa400/bpxug212.htm

Reply:
Oh a unix command. You could just use -d or --detach to run in background 🙂